RELEASE.md

```diff
@@ -1,13 +1,15 @@
-# *Infrastructure - Dependency Updates*
+# *Product Model - Forward Payout*
 
 _What is being released?_
 
-This release updates the rosetta-dsl dependency:
+This release updates the `ForwardPayout` to extend `PayoutBase` to make it consistent with all other payouts.
 
-- Version upgrades included:
-  - 8.0.0: this version is a step towards the harmonisation of the expression syntax in rules and functions. No changes are required in the CDM. See release notes at https://github.com/REGnosys/rosetta-dsl/releases/tag/8.0.0
-  - 8.1.0: this version contains various patches to improve model validation while editing the model. No changes are required in the CDM. See release notes at https://github.com/REGnosys/rosetta-dsl/releases/tag/8.1.0
+The `settlementTerms` attribute has been removed from `ForwardPayout` as it an attribute of `PayoutBase`.
 
 _Review Directions_
 
-CDM Java implementors should update their maven pom.xml to the latest CDM maven artefact (groupId com.isda, artifactId cdm) and recompile.
+In the CDM Portal, select the Textual Browser and inspect each of the changes identified above.
+
+This release does not meaningfully change any existing mapping expectations:
+
+- In the serialised JSON, the attribute ordering has changed due to the `settlementTerms` attribute change, however this has no meaningful impact on the representation of the model. 
```

Comment:
Also comment on the additional validation failures for FX samples due to their handling of payerReceiver